pkg/kube/kubehttp.go

```diff
@@ -28,6 +29,7 @@ func CreateScheme() (*apiruntime.Scheme, error) {
 	scheme := apiruntime.NewScheme()
 	builder := apiruntime.SchemeBuilder{
 		sourcev1.AddToScheme,
+		sourcev1b2.AddToScheme,
 		kustomizev2.AddToScheme,
```

Comment:
Should we add kustomize v1 here too?

Reply:
Thank you @foot 
Didn't realize that I totally miss it. I'll take a look.